Add ATLAS_PK documentation (#3349)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_ogc/server/services.rst

exception is returned.

ATLAS_PK
^^^^^^^^

This parameter allows activation of Atlas rendering by indicating which
features we want to print. In order to retrieve an atlas with all features,

> the ``*`` symbol may be used (according to the maximum number of feature

Reviewer: ```suggestion
the ``*`` symbol may be used (according to the maximum number of features
```

allowed in the project configuration).

When ``FORMAT`` is ``pdf``, a PDF document with one feature per page will be

> returned. For all other formats, only the first page is returned.

Reviewer 2: i'm confused by this behavior so, updating my comment in https://github.com/qgis/QGIS-Documentation/pull/3345#discussion_r246736836:
 
1. _For all other formats, only the **first page** is returned._
Is that possible to have a two pages layout on server? If yes, with jpg, I'd lose the second page? Am I right?

2. _"When FORMAT is pdf, a PDF document with **one feature per page** is returned."_
For me (i'm not server user but it'd be inline with the desktop) it's more a matter of file(s) per feature than page per feature. 

What I naively thought would be the behavior is:
`When FORMAT is pdf, a single PDF document combining each feature page(s) is returned. For all other formats, each feature is returned in separate file(s) per page.`

The limitation with pdf on server would be that I can not output separate pdf files for each feature?

Author: Hi @DelazJ,

> 1. For all other formats, only the first page is returned.
> Is that possible to have a two pages layout on server? If yes, with jpg, I'd lose the second page? Am I right?

Yes and yes :).

> 2. "When FORMAT is pdf, a PDF document with one feature per page is returned."
> For me (i'm not server user but it'd be inline with the desktop) it's more a matter of file(s) per feature than page per feature.

> The limitation with pdf on server would be that I can not output separate pdf files for each feature?

Actually, the server is able to return only one document per request. In the case of PDF format, the document may contain several pages.

What do you think about:

"When FORMAT is pdf, a single PDF document combining each feature page(s) is returned. For all other formats, a single page is returned."

exception is returned.

ATLAS_PK
^^^^^^^^

This parameter allows activation of Atlas rendering by indicating which
features we want to print. In order to retrieve an atlas with all features,
the ``*`` symbol may be used (according to the maximum number of features
allowed in the project configuration).

> When ``FORMAT`` is ``pdf``, a PDF document with one feature per page will be

Reviewer: ```suggestion
When ``FORMAT`` is ``pdf``, a single PDF document combining the feature pages is returned.
For all other formats, a single page is returned.
```

> returned. For all other formats, only the first page is returned.

Reviewer: ```suggestion
For all other formats, a single page is returned.
```
